docs/content/en/docs/Reference/image_matrix.md

```diff
@@ -23,21 +23,54 @@ Below is a list of the available images and their locations on the quay.io regis
 
 Base images are tagged with specific upstream versions (e.g. Ubuntu 20 LTS is pinned to Ubuntu 20:04, openSUSE to openSUSE leap 15.4, etc.).
 
```

Comment:
I would add a note here to clarify the difference between core and standard, because we lose that with this change

Reply:
@Itxaka the note is a couple of lines above, but I agree that maybe it needs to be a banner or so, WDYT?

Reply:
oooh, I didnt see that! Its good for me then :D